.gitignore

```diff
@@ -24,6 +24,8 @@
 *.handout.pdf
 *.handout.lyx
 *.synctex.gz
+*.sh
+bootstrapcensus.txt
 
 # git-lfs files cannot be included in a template repository
```

Comment:
This line and a few below seem like holdovers from the template. We can probably remove.